workshops/test/test_event.py

```diff
@@ -187,14 +187,14 @@ def test_website_url(self):
             'https://github.com/user-name/repo-name',
             'https://github.com/user-name/repo-name/',
         ]
-        WEBSITE = 'https://user-name.github.io/repo-name'
+        WEBSITE = 'https://user-name.github.io/repo-name/'
```

Comment:
Hm - do we absolutely require the trailing / ?  Do we have tests for what happens when it's not present?  Does the web interface reject URLs lacking the trailing / ?